tests/lib/path_utils.py

```diff
@@ -20,3 +22,29 @@ def expand_env_vars(obj):
     else:
         obj = os.path.expandvars(obj)
     return obj
+
+
+def get_os_name():
+    """Function for getting OS name"""
+    if sys.platform == "win32":
+        os_name = 'Windows'
+    else:
+        os_name = 'Linux'
```

Comment:
what about mac?

Reply:
Implementation will be replaced with solution from infrastructure later